categories.md

```diff
@@ -1,14 +1,15 @@
 ---
 layout: page
-title: Tags
+title: Categories
 ---
-{% assign tags =  site.recipes | map: 'tags' | join: ','  | split: ',' | uniq %}
+{% assign tags =  site.recipes | map: 'tags' | uniq | sort_natural %}
+{% assign sorted =  site.recipes | sort_natural: "title" %}
 {% for tag in tags %}
-  <h3 id="{{tag | slugify }}">{{ tag | slugify }}</h3>
+  <h3 id="{{tag | slugify }}">{{ tag | upcase}}</h3>
```

Comment:
I think it looks better upper case. not a huge deal, if it looks off i can change it back